de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -160,12 +160,255 @@ Congratulations. You have successfully set up your Git integration!
 
 == Generic Systems Engineering (Concept) Language 
 
-=== Categories and Properties
+Concepts are described using the Generic Systems Engineering Language (GSEL).
+The following gives a break-down on the key words and functionalities provided by the GSEL.
+
+=== Concepts
+
+A  _Concept_ is always the root of a concept.concept file.
+It serves as the main container of the concept.
+It also fulfills the task of carrying global meta-information.
+
+A Concept may also have the following attributes:
+
+displayname::
+This name will be shown to the user when interacting with the concept in the UI.
+For example when activating a concept, the displayname is shown.
+
+description::
+The description of the concept.
+
+version:: 
+The version number of the concept. 
+Must be given in the form of `.` separated numbers.
+For each version number, the IDE tools generate a corresponding `concept_vVERSION.xmi` in the concept folder.
+This file is used to automtically migrate old repositories based on an older concept version to the newest version.
+
+WARNING: When editing a concept, the version number should be increased before making any changes to the concept.
+Otherwise the changes will be considered to be part of the old version.
+Not doing this will cause automatic migration to fail!
 

```

Comment:
I would add some introducing words about SEs and Cs here before explainging them in the next sections. To provide some background what their main purpose is.... I guess understanding the difference between SEs and Cs is a fundamental thing that needs to be understood before all of the detailed information in the next section.

I would propose something like this:
The concept language distinguishs between Structural Elements and Categories. Structural Elements decompose the system into its basic structure, Categories contain the actual engiering data. By decoupling the engineering domain data from the system model's structure, it is possible to add new concepts to a system model later and to also open system models without all concepts loaded in Virtual Satellite.

Reply:
Included proposition. I tried to break down the last sentence a bit since it appeared as quite long.
Please check if I retained the meaning.